vault_cli/client.py

```diff
@@ -422,6 +422,8 @@ def move_secrets(
     template_prefix = "!template!"
 
     def _render_template_value(self, secret: types.JSONValue) -> types.JSONValue:
+        if isinstance(secret, dict):
+            return {k: self._render_template_value(v) for k, v in secret.items()}
```

Comment:
For the sake of it, wan we do it for lists too ?

Reply:
it will be reworked when removing the hardcoded "value" key :)